db_json_util.py

```diff
@@ -52,8 +52,30 @@ def main() -> None:
         raise FileNotFoundError(f'Database file {args.file} not found')
 
     if args.api:
-        print('using api database')
-        # TODO: implement
+        # print('using api database')
+        if args.import_data:
+            if args.json_chains and args.json_rpc_urls:
```

Comment:
I'd suggest we change this here to

```suggestion
            if args.json_chains:
```

to let this code run even if we haven't added a file for the RPC urls